src/ProductConstructionService/ProductConstructionService.Deployment/DeploymentOptions.cs

```diff
@@ -28,4 +36,26 @@ public class DeploymentOptions
     public required bool IsCi { get; init; }
     [Option("entraAppId", Required = true, HelpText = "Entra app ID")]
     public required string EntraAppId { get; init; }
+    [Option("redisConnectionString", Required = true, HelpText = "Redis Cache connection string")]
+    public required string RedisConnectionString { get; init; }
+
+    public async Task<IServiceCollection> RegisterServices(IServiceCollection services)
+    {
+        services.AddLogging(logging => logging.AddConsole());
+
+        services.AddTransient<IProcessManager>(sp => new ProcessManager(sp.GetRequiredService<ILogger<ProcessManager>>(), string.Empty));
```

Comment:
Isn't it better to set the second argument to `git`?

Reply:
It could be, but we don't use git, and it wouldn't be installed so it probably wouldn't work. Maybe a better option would be to have a constructor without that parameter, but I'm not sure if it'd break some code somewhere

Reply:
It's still better to see "executing git ..." and then failing "git not found" rather than whatever the first argument of a git call is to be evaluated as a binary

Reply:
Also git is installed in every AzDO agent, otherwise it would not be able to clone the repo for the build

Reply:
I think just `git` works